CefSharp.Core/Internals/CefTaskScheduler.h

```diff
@@ -42,7 +43,11 @@ namespace CefSharp
                 }
                 catch (Exception^)
                 {
-
+                    // This should never ever happen.
```

Comment:
Manually call AppDomain.UnhandledException handler if we can? I'm not sure we can do that. I'll take a look.